cli/test/test_runner_kubernetes.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import utils as testutils
+
+from popper.config import PopperConfig
+from popper.parser import YMLWorkflow
+from popper.runner import WorkflowRunner
+
+from popper.cli import log as log
+
+
+class TestKubernetesDockerRunner(unittest.TestCase):
+    def setUp(self):
+        log.setLevel('CRITICAL')
+
+    @unittest.skipIf(
+        os.environ['ENGINE'] != 'docker' or os.environ['RESMAN'] != 'kubernetes',
+        'ENGINE != docker or RESMAN != kubernetes')
+    def test_docker_basic_run(self):
+        repo = testutils.mk_repo()
+        conf = PopperConfig(workspace_dir=repo.working_dir)
```

Comment:
thanks a lot again @edeediong for working on this, really appreciate it! 

In order to instruct popper to use the kubernetes runner, we can need to add `resman_name='kubernetes'` to these arguments.

Reply:
Alright sure!

Reply:
done

Reply:
If I add `resman_name='kubernetes'` to `PopperConfig` as a parameter, it sorta breaks the codebase.